mlir/include/mlir/Dialect/Rock/Tuning/RockTuning.h

```diff
@@ -32,9 +32,11 @@ enum class TuningParamSetKind : uint32_t {
   // configurations that have been shown not to yield good performance.
   // (Note: this filtering is currently unimplemented).
   Full = 1,
+  // A greedy approach that explores exhaustive tuning space.
+  Greedy = 2,
   // A tuning space consisting of all possible sets of tuning parameters,
   // excluding those that could not be applicable to the given problem.
-  Exhaustive = 2,
+  Exhaustive = 3,
```

Comment:
Changing exhaustive enum value to 3, i am not sure it may end up selectin greedy if some API is passing integer value to specify tuning kind. It would  be better to keep greeey  = 3